### PR TITLE
release 6.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by alexjfisher — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,19 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-redis/compare/v6.0.0...v6.1.0)
+
+**Implemented enhancements:**
+
+- Add Ubuntu 20.04 support [\#357](https://github.com/voxpupuli/puppet-redis/pull/357) ([mmoll](https://github.com/mmoll))
+
+**Closed issues:**
+
+- Debian 10 dotdeb buster  [\#361](https://github.com/voxpupuli/puppet-redis/issues/361)
```
Comment (ekohl) [inline CHANGELOG.md:17]: Is this really relevant for the release? I'm not sure I've ever seen an issue that was relevant for a release and we should consider disabling issues altogether in GHCG.

Comment (bastelfreak) [reply CHANGELOG.md:17]: I added the skip-changelog to it and regenerated the changelog.
In general I think we should continue including issues. I often use them as a workaround. Sometimes a PR needs to fix multiple things to get CI working again. I create multiple issues and link them to the single PR.

**Comment by bastelfreak** [comment]
rebuild the changelog and this time the reference.md as well

File: CHANGELOG.md (hunk 1 above)
Comment (ekohl) [reply CHANGELOG.md]: Fair enough.

**Review by ekohl — APPROVED** [review]

**Pull request merged**